v0/destinations/google_ads_enhanced_conversion/data/trackConfig.json

```diff
@@ -0,0 +1,129 @@
+[
+  {
+    "destKey": "partialFailure",
+    "sourceKeys": "properties.partialFailure",
+    "required": true
+  },
+  {
+    "destKey": "conversionAdjustments[0].gclidDateTimePair.gclid",
+    "sourceKeys": "properties.gclid",
+    "required": false,
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "destKey": "conversionAdjustments[0].gclidDateTimePair.conversionDateTime",
+    "sourceKeys": "properties.conversionDateTime",
+    "required": false,
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "destKey": "conversionAdjustments[0].restatementValue.adjustedValue",
+    "sourceKeys": "properties.adjustedValue",
+    "required": false,
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "destKey": "conversionAdjustments[0].restatementValue.currencyCode",
+    "sourceKeys": "properties.currencyCode",
+    "required": false
+  },
+  {
+    "destKey": "conversionAdjustments[0].orderId",
+    "sourceKeys": "properties.orderId",
+    "required": true,
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "destKey": "conversionAdjustments[0].adjustmentDateTime",
+    "sourceKeys": "properties.adjustmentDateTime",
```

Comment:
```suggestion
    "sourceKeys": "properties.adjustmentTime",
```

Reply:
Also we can take `timeStamp` if the above is not present as this considers the time of adjustment

Reply:
Our timestamp format is not same as google's. Google's format is `The format is "yyyy-mm-dd hh:mm:ss+|-hh:mm", e.g. "2019-01-01 12:32:45-08:00"` and our format is `2019-01-01T11:15:18.299Z`

Reply:
I tried to use the originalTimestamp with the required format but the API is throwing `LATER_THAN_MAXIMUM_DATE` this error.

Reply:
Ok lets keep the same